dss/api/bundles/__init__.py

```diff
@@ -132,6 +133,41 @@ def get(
         response.headers['Link'] = link
         return response
 
+@dss_handler
+def enumerate(replica: str, prefix: str = 'bundles/', token: str = None,
+              per_page: int = PerPageBounds.per_page_max,
+              search_after: typing.Optional[str] = None):
+
+    if prefix and not prefix.startswith('bundles/'):
```

Comment:
Should prefix be optional?  We could ensure the user doesn't input a bad prefix.  For example, something like:

```
from dss.storage.identifiers import BUNDLE_PREFIX


def enumerate(replica, prefix=None, args, ...):
    search_prefix = f'{BUNDLE_PREFIX}/{prefix}' if prefix else f'{BUNDLE_PREFIX}/'
    ... other stuff ...

Reply:
addressed